SquareLine/gen.py

```diff
@@ -0,0 +1,232 @@
+from genericpath import isdir
```

Comment:
`genericpath` module shouldn't normally be imported. Instead you can use `os.path.isdir`